client/src/components/Atoms/StatusBar.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Divider, Classes } from '@blueprintjs/core'
+import {
+  Row,
+  // IFlexboxProps is needed by TS in order to export StatusBar
+  // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+  IFlexboxProps,
+} from './Layout'
+
+/**
+ * A container component that shows an audit status bar. Uses space-between to
+ * separate left/right children.
+ */
+export const StatusBar = styled(Row).attrs({
+  justifyContent: 'space-between',
+  className: Classes.TEXT_LARGE,
```

Comment:
I've found that the attributes provided to `attrs` aren't type checked. I haven't had a chance to dig into a fix but thought I'd flag 🎏

Reply:
Oh, dang, that's annoying! Thanks for the heads up